arch/arm/boot/dts/overlays/README

```diff
@@ -1873,6 +1873,42 @@ Params: panel                   Display panel (required):
         speed                   Display SPI bus speed
 
 
+Name:   pca953x
+Info:   TI PCA953x family of I2C GPIO expanders. Default is for NXP PCA9534.
+Load:   dtoverlay=pca953x,<param>=<val>
+Params: addr                    I2C address of expander. Default 0x20.
+        pca6416                 Select the NXP PCA6416 (16 bit)
```

Comment:
@pelwell Is there meant to be a blank line between parameters or not? Some overlays do, some don't.

Reply:
I leave that to the author - longer descriptions would probably warrant a blank line, otherwise it's not necessary.

Reply:
I'll leave it without then.